clang/test/SemaCXX/constant-expression-cxx1z.cpp

```diff
@@ -177,3 +177,16 @@ namespace LambdaCallOp {
     p();
   }
 }
+
+/// This used to crash due to an assertion failure,
+/// see gh#67690
```

Comment:
NIT, why three slashes? 
```suggestion
// This used to crash due to an assertion failure,
// see gh#67690
```

Reply:
Because it's not a RUN or expected-* line.